tests/pr_test.go

```diff
@@ -141,6 +142,17 @@ func TestRunStandardSolutionSchematics(t *testing.T) {
 		},
 	}
 
+	serviceCredentialNames := map[string]string{
+		"admin": "Administrator",
+		"user1": "Viewer",
+		"user2": "Editor",
+	}
+
+	serviceCredentialNamesJSON, err := json.Marshal(serviceCredentialNames)
+	if err != nil {
+		log.Fatalf("Error converting to JSON: %s", err)
+	}
+
```

Comment:
This is different to elasticsearch and seems to be better. Should elasticsearch DA test be improved like this? If yes, can you raise an issue to explain what it does.

Reply:
I will create an issue in elasticsearch for this.